### PR TITLE
feat: Allow to enable/disable Cover Art in Web App & load from filesystem

### PR DESCRIPTION
**Pull request by pabera**
* [x] feat: In certain situations it might be useful to disable cover art in the web app (especially when there is a lot music)
  * [x] Allow Setting to be managed in Web App
* [x] feat: Try reading cover art from file system (`[Cc]over.[jpg|png]`) if it does not exist in audio file
  * [x] Update Documentation
* [x] fix: fixes a bug where audio files pretended to have a cover art, but actually didn't.
* [x] Double check uncommon file type support (flac, m4a, ogg)

### CODE REVIEW

File: src/webapp/src/commands/index.js [hunk 1]
```diff
@@ -120,6 +120,7 @@ const commands = {
     _package: 'volume',
     plugin: 'ctrl',
     method: 'set_volume',
+    argKeys: ['volume'],
```
Comment (s-martin) [inline src/webapp/src/commands/index.js:123]: Unrelated to this change?

Comment (pabera) [reply src/webapp/src/commands/index.js:123]: yes. didn't want to create a separate PR for it

**Review by s-martin — APPROVED** [review]

**Comment by s-martin** [comment]
I like the possibility to get covers also from the file system, because that eases migration from V2.

**Comment by s-martin** [comment]
Do you also want to integrate the recently requested support of covers embedded in flac or m4a or should that be a different PR?

**Comment by pabera** [comment]
> Do you also want to integrate the recently requested support of covers embedded in flac or m4a or should that be a different PR?

Should already be covered

**Pull request merged**